src/test/features/response/routes/check-and-send.ts

```diff
@@ -191,20 +190,6 @@ describe('Defendant response: check and send page', () => {
                 .toLocation(ResponsePaths.counterClaimPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })))
           })
 
-          it('should redirect to partial-admission handoff page when defendant response is part admission', async () => {
```

Comment:
What happens now?

Reply:
I guess it falls under "when everything is fine" now :wink:

Jokes aside, that's something I'd like to add. But the way these tests are structured, it doesn't feel like there's a good place for it, because there is no explicit test for defence response (when not counter-claiming) either.

Reply:
We should have the test documenting the behaviour of when user submits partial defence. The expected outcome is to take user to the confirmation page.

When test case is missing feel free to add it either now as this will be just few lines to separate PR.

Reply:
Updated PR with a new test added